homeassistant/components/goalzero/config_flow.py

```diff
@@ -34,19 +34,20 @@ async def async_step_user(self, user_input=None):
 
             try:
                 await self._async_try_connect(host)
-                return self.async_create_entry(
-                    title=name,
-                    data={CONF_HOST: host, CONF_NAME: name},
-                )
             except exceptions.ConnectError:
                 errors["base"] = "cannot_connect"
-                _LOGGER.exception("Error connecting to device at %s", host)
+                _LOGGER.error("Error connecting to device at %s", host)
             except exceptions.InvalidHost:
                 errors["base"] = "invalid_host"
                 _LOGGER.exception("Invalid data received from device at %s", host)
```

Comment:
We only log unknown errors with exception.
```suggestion
                _LOGGER.error("Invalid data received from device at %s", host)
```

Reply:
Is the log message correct? The exception is invalid host but the log says that the data received from the host is invalid. The error message in the frontend says "Invalid host provided". What does invalid host mean?

Reply:
Any host that is not a yeti. If a host responds with a json object, the pypi package should raise exception if the received json does not match the expected keys.